cdap-app-fabric/src/main/java/io/cdap/cdap/app/preview/PreviewStatus.java

```diff
@@ -51,18 +51,20 @@ public boolean isEndState() {
 
   private final Status status;
   private final BasicThrowable throwable;
+  private final Long submitTime;
   private final Long startTime;
   private final Long endTime;
   private final Integer positionInWaitingQueue;
 
-  public PreviewStatus(Status status, @Nullable BasicThrowable throwable, @Nullable Long startTime,
+  public PreviewStatus(Status status, Long submitTime, @Nullable BasicThrowable throwable, @Nullable Long startTime,
```

Comment:
Why use `Long` instead of the primitive `long`?

Reply:
done.